### PR TITLE
Created segment navigation for mobile and desktop view

### PR DESCRIPTION
**Pull request by zeeshanak23**
Screenshot (145)
Screenshot (147)
Screenshot (146)

### CODE REVIEW

File: marketing/src/sections/navigation/SegmentNavigation/SegmentNav.css [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+.segment-Navigation .features>a {
+    font-weight: 500;
+    color: black;
+    margin: 0px 60px;
+    justify-content: space-between;
+    text-decoration: none;
+    font-size: 1.6rem;
+}
+
+.segment-Navigation .Nav-Btn>a {
```
Comment (amezng) [inline marketing/src/sections/navigation/SegmentNavigation/SegmentNav.css:36]: Here don't use capital letter

File: marketing/src/sections/navigation/SegmentNavigation/SegmentNav.css [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    font-weight: 500;
+    color: black;
+    margin: 0px 60px;
+    justify-content: space-between;
+    text-decoration: none;
+    font-size: 1.6rem;
+}
+
+.segment-Navigation .Nav-Btn>button {
```
Comment (amezng) [inline marketing/src/sections/navigation/SegmentNavigation/SegmentNav.css:45]: Use small letters like `nav-btn`

File: marketing/src/sections/navigation/SegmentNavigation/SegmentNav.jsx [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                    <span>
+                        <h1>Twilio Engage</h1>
+                        <p>Personalized and real-time customer experience</p>
+                    </span>
+                </div>
+                <div className='feature-menu-list'>
+                    <img src="./img/sections/navigations/SegmentNav/c2.svg" alt="" />
+                    <span>
+                        <h1>Twilio Engage</h1>
+                        <p>Personalized and real-time customer experience</p>
```
Comment (amezng) [inline marketing/src/sections/navigation/SegmentNavigation/SegmentNav.jsx:85]: Don't repeat text. Use text from lorem ipsum

File: marketing/src/sections/navigation/SegmentNavigation/SegmentProductMenu.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+const SegmentProductMenu = [
```
Comment (amezng) [inline marketing/src/sections/navigation/SegmentNavigation/SegmentProductMenu.js:1]: use small letter in the beginning for declaring constants

**Review by amezng — CHANGES_REQUESTED** [review]

**Comment by zeeshanak23** [comment]
Done

File: marketing/src/index.js [hunk 5]
```diff
@@ -2,10 +2,12 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import reportWebVitals from "./reportWebVitals";
+import SegmentNav from "./sections/navigation/SegmentNavigation/SegmentNav";
```
Comment (amezng) [inline marketing/src/index.js:5]: Don't change this file ever

File: marketing/src/sections/navigation/SegmentNavigation/SegmentNav.css [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+    display: flex;
+    justify-content: space-between;
+}
+
+.productBtn h1 {
+    font-size: 1.6rem;
+    font-weight: 300;
+}
+
+.productMenu>div p {
```
Comment (amezng) [inline marketing/src/sections/navigation/SegmentNavigation/SegmentNav.css:116]: Use context classes everywhere

File: marketing/src/sections/navigation/SegmentNavigation/SegmentNav.jsx [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+                                                    </div>
+                                                </>
+                                            )
+                                        })}
+                                        <div>
+                                            <h1>Products</h1>
+                                            <div >
+                                                {productBarMenu.map((t, i) => {
+                                                    return (
+                                                        <>
```
Comment (amezng) [inline marketing/src/sections/navigation/SegmentNavigation/SegmentNav.jsx:49]: No need to use `<>` empty tag as there is already a div

File: marketing/src/sections/navigation/SegmentNavigation/SegmentNav.jsx [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+                                        </div>
+
+                                    </div>
+
+                                    <div className='items'>
+                                        <span>FEATURES</span>
+                                        <div className='secondBar'>
+                                            {featureProduct.map((t, i) => {
+                                                return (
+                                                    <>
```
Comment (amezng) [inline marketing/src/sections/navigation/SegmentNavigation/SegmentNav.jsx:73]: Similary here you don't need empty tag

File: marketing/src/sections/navigation/SegmentNavigation/SegmentResponsive.css [hunk 9]
```diff
@@ -0,0 +1,106 @@
+@media only screen and (max-width:415px) {
+    .segment-Navigation .features>a {
+        display: none;
```
Comment (amezng) [inline marketing/src/sections/navigation/SegmentNavigation/SegmentResponsive.css:3]: Why using capital `N` here ?

File: marketing/src/sections/navigation/SegmentNavigation/SegmentResponsive.css [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+    .mobile-product-list>div>span {
+        font-weight: 500;
+
+    }
+
+    .product-arrow-menu {
+        float: right;
+        margin-left: 310px;
+
+    }
```
Comment (amezng) [inline marketing/src/sections/navigation/SegmentNavigation/SegmentResponsive.css:64]: Use context class everywhere

**Review by amezng — CHANGES_REQUESTED** [review]

**Comment by amezng** [comment]
This is not as per the design

**Comment by amezng** [comment]
text color is not good. fix it

**Comment by amezng** [comment]
There is no change to this PR, after my comment. Fix it ASAP.

File: marketing/src/index.js [hunk 11]
```diff
@@ -1,11 +1,13 @@
...
 import ReactDOM from "react-dom/client";
-import "./index.css";
 import reportWebVitals from "./reportWebVitals";
 
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
-root.render(<React.StrictMode></React.StrictMode>);
+root.render(
+    <React.StrictMode>
+    </React.StrictMode>
```
Comment (amezng) [inline marketing/src/index.js:9]: Don't change this file.

**Review by amezng — APPROVED** [review]

**Pull request merged**